### PR TITLE
Personas

### PR DESCRIPTION
**Pull request by MarianRaphael**
## Description

Initial selection of personas and transfer from FF Google Drive. 
Creating a Personas folder in the Product area. Deleting / moving the current personas from the Design folder.

### CODE REVIEW

File: product/personas/readme.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Personas
```
Comment (ZJvandeWeg) [inline product/personas/readme.md:1]: Nitpick: Can we rename this file to README.md for consistency?

**Comment by ZJvandeWeg** [comment]
@joepavitt Can you help with reviewing?

File: product/personas/readme.md (hunk 1 above)
Comment (MarianRapahel) [reply product/personas/readme.md]: Done

**Comment by joepavitt** [comment]
@MarianRapahel have you seen the details of our Personas in Figma? Not just what was in our Handbook?

**Comment by MarianRapahel** [comment]
> @MarianRapahel have you seen the details of our [Personas in Figma](https://www.figma.com/file/qOKeqGBpDyzf6ay5BL1Lgp/User-Personas?node-id=0%3A1&t=qwPzFTiKbYzQ5nFo-1)? Not just what was in our Handbook? Just want to know for context when I'm reviewing this. I like some of the additional questions/details you've put in here for a given Persona. Just want to try and keep some of the formatting if we can, which is lost at the moment.

@joepavitt No, didn't know the personas in Figma (and the FlowForge Figma space in general not) yet. Good to know, yes the content must definitely be included. I would also enter the new personas in Figma afterwards and remove them from the Google Drive Space. For ideation and further development of the personas, Figma is more suitable.

**Comment by joepavitt** [comment]
> I would also enter the new personas in Figma afterwards and remove them from the Google Drive Space

Not sure I was even aware of them existing in Google Drive! Figma has been my working space for the Personas, and where we've focussed the output from Workshops. Happy to transfer them over to there once we've settled on these.

I've sent you an invite into the FlowForge Figma team, happy to talk you through some of the content on there as and when suits.

I will do some work tomorrow to make sure the content consistent between our existing personas (in Figma) and the content you've added here. 

I'd like to make some visual improvements to the personas too, but can do that as a follow-up PR as I don't want to hold this up and visuals are a nice-to-have anyway.

File: product/personas/Danielle_Developer.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+Age: 24
+
+Job Level: Individual Contributor
+
+Software Skillset: Computer Science & AI, BSc
+
+### Job To Be Done:
+
+- Software Developer in New Startup Inc. (works for [Sahib](./Sahib Startup CTO.md))
```
Comment (ZJvandeWeg) [inline product/personas/Danielle_Developer.md:13]: Does this link work? It's missing underscores

File: product/personas/Peter_PLC_Engineer.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+- Having a satisfied team and highly motivated employees
+
+- establishing "IIoT" (instruction from the boss)
+
+### Gains Information by:
+
+- LinkedIn
+
+- Trainings
```
Comment (ZJvandeWeg) [inline product/personas/Peter_PLC_Engineer.md:35]: Trade magazines in the cafeteria?

**Comment by ZJvandeWeg** [comment]
Looks good to me, if the links work. I'd be surprised, but markdown is surprising.

**Comment by joepavitt** [comment]
@MarianRaphael for the "Purchasing Role" - what scale are your numbers? Monthly or Annually?

**Comment by joepavitt** [comment]
Okay, so to get these personas consistent with the others (I've now written them up in Figma), I need the following:

- Interests (not just professional interests)
- Goals (can crossover with motivations a little)
- Needs & expectations (generally professionally-focussed)

for Peter, Paul & Isabella please.

I also need to get the Bio/"Job To Be Done" parts into Figma too, and will need to write equivalents for our existing persona sets.

File: product/personas/Peter_PLC_Engineer.md (hunk 3 above)
Comment (joepavitt) [reply product/personas/Peter_PLC_Engineer.md:35]: Have added this into Figma

File: product/personas/Danielle_Developer.md (hunk 2 above)
Comment (joepavitt) [reply product/personas/Danielle_Developer.md:13]: There are a few technical issues with the way they're written, some clean up is required, e.g. images don't render as they are at the moment either. Plan on sweeping through and fixing these issues once we have consistent personas in Figma.

**Comment by MarianRaphael** [comment]
> Okay, so to get these personas consistent with the others ([I've now written them up in Figma](https://www.figma.com/file/qOKeqGBpDyzf6ay5BL1Lgp/User-Personas?node-id=0%3A1&t=FO468FrYia6XZviu-1)), I need the following:
> 
> * Node-RED Experience/Awareness
> * Interests (not just professional interests)
> * Goals (can crossover with motivations a little)
> * Needs & expectations (generally professionally-focussed)
> 
> for Peter, Paul & Isabella please.
> 
> I also need to get the Bio/"Job To Be Done" parts into Figma too, and will need to write equivalents for our existing persona sets.

Done and ready for review in figma.

**Comment by ZJvandeWeg** [comment]
@joepavitt Can you get this merged?

**Comment by jdp1g09** [comment]
I still haven't had the chance to get these in, hoping to do it this week once i've got the Instance Prototyping stuff done for Pez's dev work

**Comment by joepavitt** [comment]
Have finally got round to moving all personas to a consistent template, Figma is also up to date with all of the other Personas we have, but haven't yet published.

Moved the personas into a single file as felt it was easier to scan through, although we can change that again in the future if required.

**Review by joepavitt — APPROVED** [review]

**Pull request merged**